### PR TITLE
Dandole estilos CSS al header

### PR DESCRIPTION
**Pull request by karlaojeda**

### CODE REVIEW

File: sitio_web/planes/index.html [hunk 1]
```diff
@@ -13,6 +13,20 @@
...
+    <header>
+      <nav class="nav">
+        <div class="nav__container">
+          <h2 class="nav__title">uHost</h2>
+        </div>
+        <div class="nav__list-container">
+          <ul class="nav__list">
+            <li class="nav__items">Packages</li>
+            <li class="nav__items">Customers</li>
+            <li class="nav__items nav__items--highlight">Star hosting</li>
```
Comment (plbryant) [inline sitio_web/planes/index.html:25]: "nav__items" class should be "nav__item" in singular form.

File: sitio_web/planes/main.css [hunk 2]
```diff
@@ -6,4 +6,54 @@
...
+    color: #11862f;
+}
+
+.nav__list-container {
+    display: inline-block;
+    width: calc(100% - 95px);
+
+}
+
+.nav__list-container .nav__items--highlight {
```
Comment (plbryant) [inline sitio_web/planes/main.css:34]: This should be a class modifier, not a block modifier.

**Review by plbryant — CHANGES_REQUESTED** [review]
Two errors to resolve.

File: sitio_web/planes/index.html (hunk 1 above)
Comment (karlaojeda) [reply sitio_web/planes/index.html]: ok,listo

File: sitio_web/planes/main.css (hunk 2 above)
Comment (karlaojeda) [reply sitio_web/planes/main.css]: listo, gracias.

**Review by plbryant — APPROVED** [review]

**Pull request merged**